Formula/rack.rb

```diff
@@ -23,9 +23,16 @@ def install
     rackpath.install Dir["{*,.??*}"]
 
     cd rackpath do
+      # This is a slightly grim hack to handle the weird logic around
+      # deciding whether to add a = or not on the ldflags, as mandated
+      # by Go 1.7+.
+      # https://github.com/rackspace/rack/issues/446
+      inreplace "script/build", "go1.5", "go#{Formula["go"].version}"
```

Comment:
Aye, I was just trying to not bork it for `devel`. `version` is nice because it works whatever you have installed. The upstream build script doesn't seem to have contemplated that there would be a Go beyond 1.5; it does a simple grep to see if we need the `=` or not.